Ginger/GingerCoreNETUnitTest/RunTestslib/CLITest.cs

```diff
@@ -136,15 +146,20 @@ public void CLIBadParams()
                 CLIProcessor CLI = new CLIProcessor();
 
                 // Act            
-                CLI.ExecuteArgs(new string[] { "--blabla" });
+               
+
+                Task.Run(async () =>
+                {
+                    await CLI.ExecuteArgs(new string[] { "--blabla" });
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5632404)